docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: Database Access with Oracle (Preview)
+description: How to configure Teleport Database Access with Oracle.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Oracle Server is currently in Preview mode.
+</Admonition>
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["cloud", "enterprise"]}
+  scopeOnly={true}
+  min="12.3"
+>
+  Database access for Oracle is available starting from Teleport `v12.3`.
+</Details>
+
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up Teleport to access your self-hosted Oracle server 18c or later.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Redis Self-Hosted](../../../img/database-access/guides/oracle_selfhosted.png)
```

Comment:
Should we mention somewhere this is an enterprise feature? What should we show here of `oss`?

nit: update `[Teleport Database Access Redis Self-Hosted]`